front-end/src/components/User.js

```diff
@@ -28,40 +31,42 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const rows = [
-  { secret: 'sample secret 1', dateAdded: '1/19/21' },
-  { secret: 'sample secret 2', dateAdded: '6/7/20' },
-];
-
-export default function User() {
+const User = ({ userId }) => {
   const classes = useStyles();
+  // const [rows, setRows] = useState([]);
+  let rows;
+
+  axios
+    .get(`${process.env.REACT_APP_BACKEND_SERVICE}/user/${userId}`)
```

Comment:
Do we need a `config` parameter here for the headers? We may not, I'm not experienced enough with axios to know if we can omit them